modules/core/src/v2/baseCoin.ts

```diff
@@ -402,7 +402,7 @@ export abstract class BaseCoin {
 
   /**
    * @param params
-   * @return true iff address is a wallet address. Must return false if address is outside wallet.
+   * @return true if address is a wallet address. Must return false if address is outside wallet.
```

Comment:
iff is short for "if and only if": https://en.m.wikipedia.org/wiki/If_and_only_if

Reply:
Oh my bad, sorry.